tools/importer-rest-api-specs/components/parser/commonschema/custom_field_identity_system_or_user_assigned_map.go

```diff
@@ -107,6 +107,10 @@ func (systemOrUserAssignedIdentityMapMatcher) IsMatch(_ models.FieldDetails, def
 			continue
 		}
 
+		if strings.EqualFold(fieldName, "DelegatedResources") {
```

Comment:
can we add a little context for this:

```suggestion
		// Per the API Definition, the `DelegatedResources` field is for `internal use only` - therefore we should ignore this if it's present
		if strings.EqualFold(fieldName, "DelegatedResources") {
```